modules/imaging_uploader/test/TestPlan.md

```diff
@@ -109,3 +109,28 @@
 	archive, answer 'Yes'. Check that this reupload fails with error code 4 (and not 2). 
 	Related to Redmine#14093 and PR#3555.
 	[Manual Testing]
+20. First, set the config setting 'ImagingUploader auto launch' to 'No'. Upload an MRI archive that can successfully be
+    uploaded. Check that the value of the Progress column for the uploaded archive is 'Not started'.
+    [Manual Testing]
+21. First, set the config setting 'ImagingUploader auto launch' to 'Yes'. Upload an MRI archive that can successfully be
+    uploaded and that will be processed successfully by the pipeline. Once the upload is finished, check that the value of 
+    the Progress column for the uploaded archive is 'In Progress...'
+    [Manual Testing]
+22. First, set the config setting 'ImagingUploader auto launch' to 'Yes'. Upload an MRI archive that can successfully be
+    uploaded and that will be processed successfully by the pipeline. Once the upload is finished, click on the uploaded
+    archive in the results table and select 'Detailed' in the combo box of the log panel. Check that an increasing serie of
+    dots is displayed along with a rotating bar to indicate the the archive is being processed.
+    [Manual Testing]
+23. Set the config setting 'ImagingUploader auto launch' to 'Yes'. Upload a file that can successfully be uploaded but that 
+    will fail the processing done by the MRI pipeline. After the upload succeeds, wait for the MRI pipeline to finish processing
+    the archive and check that the Progress column is set to 'Failure'. Note down the text that is displayed in the log panel
+    (Detailed view). Now change the config setting 'ImagingUploader auto launch' to 'No'. Upload the same file again. once the
+    upload is finished, make sure that the detailed log messages seen earlier are gone and have been replaced by a message that
```

Comment:
```suggestion
    upload is finished, ensure that the detailed log messages seen earlier are gone and have been replaced by a message that
```

Reply:
I changed all occurrences of 'make sure' to 'ensure'.